translations/es/pack/mut_gen_encounter.json

```diff
@@ -676,120 +622,121 @@
 	{
 		"code": "32146",
 		"name": "Centinela tipo M",
-		"text": "Guardia.\n<b>Cuando se derrote esta carta:</b> dale a Magneto una carta de Estado /*Duro/*.",
+		"text": "Guardia.\n<b>Cuando se derrote esta carta:</b> dale a Magneto una carta de Estado \"Duro\".\n<hr />\n[star] <b>Aumento</b>: dale a Magneto una carta de Estado \"Duro\" y una carta de aumento boca abajo.",
 		"traits": "Centinela."
 	},
 	{
 		"code": "32147",
 		"name": "Casco de Magneto",
-		"text": "Vincula esta carta a Magneto.\nMagneto no puede quedar confundido.",
+		"text": "Vincula esta carta a Magneto.\nMagneto no puede quedar confundido.\n<b>Respuesta de Héroe:</b> después de que tu Héroe realice un ataque básico contra Magneto, gasta recursos [energy][mental][physical] \u2192 descarta esta carta.",
 		"traits": "Armadura. Magnético."
 	},
 	{
 		"code": "32148",
 		"name": "Magneto's Armadura",
-		"text": "Vincula esta carta a Magneto.\nMagneto no puede quedar aturdido.",
+		"text": "Vincula esta carta a Magneto.\nMagneto no puede quedar aturdido.\n<b>Respuesta de Héroe:</b> después de que tu Héroe realice un ataque básico contra Magneto, gasta recursos [energy][mental][physical] \u2192 descarta esta carta.",
 		"traits": "Armadura. Magnético."
 	},
 	{
 		"code": "32149",
 		"name": "Burbuja Magnética",
-		"text": "Vincula esta carta a Magneto.\nMagneto gana Represalia 1.",
+		"text": "Vincula esta carta a Magneto.\nMagneto gana Represalia 1.\n<b>Interrupción obligada:</b> cuando Magneto vaya a sufrir cualquier cantidad de Daño, en vez de eso coloca ese Daño sobre esta carta. Luego, si hay al menos 8 de Daño aquí, descarta esta carta.",
 		"traits": "Magnético."
 	},
 	{
 		"code": "32150",
+		"flavor": "Vaya, esto no es bueno —Arcángel"
```

Comment:
Failure says
```
/home/runner/work/marvelsdb-json-data/marvelsdb-json-data/translations/es/pack/mut_gen_encounter.json: File is not valid JSON.
Expecting ',' delimiter: line 649 column 3 (char 39424)
```

It looks like a comma is missing at the end of this line.

Reply:
in my file have the comma and the las t update is update that coma xD

Reply:
correct now again and working...okey xD